src/pybind/mgr/devicehealth/module.py

```diff
@@ -118,6 +130,12 @@ def handle_command(self, _, cmd):
             return self.show_device_metrics(cmd['devid'], cmd.get('sample'))
         elif cmd['prefix'] == 'device check-health':
             return self.check_health()
+        elif cmd['prefix'] == 'device monitoring on':
+            self.set_config('enable_monitoring', 'true')
+            return 0, '', ''
+        elif cmd['prefix'] == 'device monitoring off':
+            self.set_config('enable_monitoring', 'false')
```

Comment:
Since we only check_health when monitoring is enabled, need to clear the health here in order to avoid stuck alerts